core/src/ledger/storage/wl_storage.rs

```diff
@@ -215,13 +216,13 @@ where
             self.storage.next_epoch_min_start_height =
                 height + min_num_of_blocks;
             self.storage.next_epoch_min_start_time = time + min_duration;
+
             // TODO put this into PoS parameters and pass it to tendermint
             // `consensus_params` on `InitChain` and `EndBlock`
```

Comment:
@tzemanovic how should we pass this to `FinalizeBlock`, assuming we only want to do that if the value of the param changes? In this case, it would happen thru a gov proposal right? Would we need to read the value in `finalize_block` before an `execute_governance_proposals`, then read it again to see if it changed, and then if so include it in `FinalizeBlock::consensus_param_updates`?

Reply:
Yeah, it can be changed via gov, but we could probably just always read it and pass it to Comet and if it's the same it wouldn't have any effect